Source/WebCore/rendering/svg/RenderSVGText.cpp

```diff
@@ -940,17 +941,17 @@ FloatRect RenderSVGText::repaintRectInLocalCoordinates(RepaintRectCalculation re
     if (document().settings().layerBasedSVGEngineEnabled()) {
         auto repaintRect = SVGBoundingBoxComputation::computeRepaintBoundingBox(*this);
 
-        if (const auto* textShadow = style().textShadow())
-            textShadow->adjustRectForShadow(repaintRect);
+        if (const auto& textShadow = style().textShadow(); !textShadow.isEmpty())
```

Comment:
```suggestion
        if (auto& textShadow = style().textShadow(); !textShadow.isEmpty())
```
Not sure the `const` does a lot of good in this and many other cases. Wish this was a language where `const` was default and `mutable` was the wordier case.